book/src/libs/wasp/benchspy/loki_dillema.md

```diff
@@ -0,0 +1,35 @@
+# BenchSpy - To Loki or Not to Loki?
+
+You might be wondering whether to use the `Loki` or `Direct` query executor if all you need are basic latency metrics.
+
+## Rule of Thumb
+
+If all you need is a single number, such as the median latency or error rate, and you're not interested in:
```

Comment:
I'd mention "Direct" here not at the bottom of bullets - easier "to load to the brain"